test/tools/isoltest.cpp

```diff
@@ -99,8 +99,6 @@ void SyntaxTestTool::printContract() const
 		for (auto const& error: m_test->errorList())
 			if (error.locationStart >= 0 && error.locationEnd >= 0)
 			{
-				assert(static_cast<size_t>(error.locationStart) <= source.length());
-				assert(static_cast<size_t>(error.locationEnd) <= source.length());
```

Comment:
This fails if the error is in the end of the file, that is, if the contract is incomplete.

Reply:
It would still be good to have these. What are the exact numbers? This could be a bug in the parser or scanner.

Reply:
I really don't know what happened, but I just ran it now to get the numbers and it didn't crash :confused: